alea/models/blueice_extended_model.py

```diff
@@ -48,6 +50,9 @@ def __init__(self, parameter_definition: dict, likelihood_config: dict, **kwargs
         super().__init__(parameter_definition=parameter_definition, **kwargs)
         self._likelihood = self._build_ll_from_config(likelihood_config)
         self.likelihood_names = [t["name"] for t in likelihood_config["likelihood_terms"]]
+        self.livetime_parameter_names = [t.get("livetime_parameter", None) for t in
+                                         likelihood_config["likelihood_terms"]]
+        self.livetime_parameter_names += [None]  # ancillary likelihood
         self.likelihood_names.append("ancillary_likelihood")
```

Comment:
Maybe move this line to below `self.likelihood_names = [t["name"] for t in likelihood_config["likelihood_terms"]]`.

Reply:
isn't it below?

Reply:
```
        self.likelihood_names = [t["name"] for t in likelihood_config["likelihood_terms"]]
        self.likelihood_names.append("ancillary_likelihood")
        self.livetime_parameter_names = [t.get("livetime_parameter", None) for t in
                                         likelihood_config["likelihood_terms"]]
        self.livetime_parameter_names += [None]  # ancillary likelihood
```

Reply:
ah sure. Wish it showed the line you commented on.